src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1117,6 +1117,7 @@ protected override void GetItem(string path)
 
             try
             {
+#if !UNIX
                 bool retrieveStreams = false;
                 FileSystemProviderGetItemDynamicParameters dynamicParameters = null;
```

Comment:
Once we stop providing the `-Stream` dynamic parameter for FileSystemProvider on Unix, we don't need to add so many `#if UNIX/endif` blocks in this file because those codes will not be hit. I think that will make the code more readable and easy to maintain.